std/path.d

```diff
@@ -2738,17 +2738,23 @@ else version (Posix)
     See_Also:
         $(LREF asAbsolutePath) which does not allocate
 */
-string absolutePath(string path, lazy string base = getcwd())
+string absolutePath(string path, string base)
     @safe pure
 {
     import std.array : array;
-    if (path.empty)  return null;
-    if (isAbsolute(path))  return path;
```

Comment:
I don't think you can remove these here. It changes the behavior of this overload.